client/state/posts/reducer.js

```diff
@@ -282,6 +281,23 @@ export function edits( state = {}, action ) {
 				[ action.siteId ]: omit( state[ action.siteId ], action.postId || '' )
 			} );
 
+		case POST_SAVE_SUCCESS:
+			if ( ! state.hasOwnProperty( action.siteId ) || ! action.savedPost ) {
+				break;
+			}
+			const { postId, siteId, savedPost } = action;
+
+			if ( postId ) {
+				return state;
```

Comment:
Also, should we make this so specific to the empty postId case? Presumably if it were ever the case that the ID from the savedPost differs from the `postId` we sent with the save, we'd want to re-key?